gade.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Jan  5 12:32:12 2020
+
+@author: rooga
+"""
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+def feature_extract() :

```

Comment:
I think you meant to receive a dataframe object as 'data'. I'm going to just add that.